airflow/jobs/scheduler_job.py

```diff
@@ -161,12 +165,16 @@ def __init__(
         # Check what SQL backend we use
         sql_conn: str = conf.get_mandatory_value("database", "sql_alchemy_conn").lower()
         self.using_sqlite = sql_conn.startswith("sqlite")
-        # Dag Processor agent - not used in Dag Processor standalone mode.
+        # Dag Processor agent - not used in Dag Processor eartstandalone mode.
```

Comment:
```suggestion
        # Dag Processor agent - not used in Dag Processor standalone mode.
```

I figure this wasn't intentional.

Reply:
Why do you think so :) ?